src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/ProxyStreamFactoryBuilder.java

```diff
@@ -47,6 +47,7 @@
     private LongSupplier supplyCorrelationId;
     private Slab bufferPool;
     private Cache cache;
+    private BudgetManager budgetManager;
```

Comment:
Do we need to store `BudgetManager` here or can we reference as member of `Cache` instead?

Reply:
We use BudgetManager in ProxyConnectReplyStream. Perhaps it can be accessed from request.cache.budgetManager, but this seems better